src/algorithm/REDUCE_SUM-OMPTarget.cpp

```diff
@@ -56,62 +56,30 @@ void REDUCE_SUM::runOpenMPTargetVariant(VariantID vid, size_t tune_idx)
 
   } else if ( vid == RAJA_OpenMPTarget ) {
 
-    if (tune_idx == 0) {
-
-      startTimer();
-      for (RepIndex_type irep = 0; irep < run_reps; ++irep) {
-
-        RAJA::ReduceSum<RAJA::omp_target_reduce, Real_type> sum(m_sum_init);
-
-        RAJA::forall<RAJA::omp_target_parallel_for_exec<threads_per_team>>(
-          RAJA::RangeSegment(ibegin, iend),
-          [=](Index_type i) {
-            REDUCE_SUM_BODY;
-        });
-
-        m_sum = sum.get();
-
-      }
-      stopTimer();
-
-    } else if (tune_idx == 1) {
-
-      startTimer();
-      for (RepIndex_type irep = 0; irep < run_reps; ++irep) {
-
-        Real_type tsum = m_sum_init;
+    startTimer();
+    for (RepIndex_type irep = 0; irep < run_reps; ++irep) {
 
-        RAJA::forall<RAJA::omp_target_parallel_for_exec<threads_per_team>>(
-          RAJA::RangeSegment(ibegin, iend),
-          RAJA::expt::Reduce<RAJA::operators::plus>(&tsum),
-          [=] (Index_type i, Real_type& sum) {
-            REDUCE_SUM_BODY;
-          }
-        );
+      Real_type tsum = m_sum_init;
 
-        m_sum = static_cast<Real_type>(tsum);
+      RAJA::forall<RAJA::omp_target_parallel_for_exec<threads_per_team>>(
+        RAJA::RangeSegment(ibegin, iend),
+        RAJA::expt::Reduce<RAJA::operators::plus>(&tsum),
+        [=] (Index_type i, Real_type& sum) {
```

Comment:
`Real_type& sum` would need to be `RAJA::expt::ValOp<Real_type, RAJA::operators::plus> & sum` if we are pulling in the latest changes from RAJA/develop.

Reply:
We are not on RAJA develop yet